### DISCUSSION TITLE
Add fontawesome into Tailwind.

### DISCUSSION THREAD

**Discussion by JoeyCodes97** [tailwindlabs/tailwindcss#7359] [Help]
Good evening, can someone please explain to Fontawesome into a tailwind project? I used NPM to install it but after that Im not sure. I added an icon but it it does not show on the page. Am I missing a step here? PLease advise, thank you.

**Comment by Andre-ADPC** [comment]
Hi, @JoeyCodes97  I was looking for the same solution and found [this repo](https://github.com/mattpfeffer/tailwind-fontawesome) to be a working solution. The package can be installed via NPM with: `npm install tailwind-fontawesome` or using Yarn: `yarn add tailwind-fontawesome`. If it is too late for you, maybe it can help someone else.
